persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/impl/table/AbstractTableManager.java

```diff
@@ -21,36 +21,62 @@
  */
 public abstract class AbstractTableManager implements TableManager {
 
+   private static final String DEFAULT_IDENTIFIER_QUOTE_STRING = "\"";
+
    private final Log log;
    protected final ConnectionFactory connectionFactory;
    protected final TableManipulationConfiguration config;
    protected final String timestampIndexExt = "timestamp_index";
    protected final String segmentIndexExt = "segment_index";
 
-   protected String identifierQuoteString = "\"";
-   protected String cacheName;
-   protected DbMetaData metaData;
-   protected TableName tableName;
-
-   protected String insertRowSql;
-   protected String updateRowSql;
-   protected String upsertRowSql;
-   protected String selectRowSql;
-   protected String selectMultipleRowSql;
-   protected String selectIdRowSql;
-   protected String deleteRowSql;
-   protected String loadAllRowsSql;
-   protected String countRowsSql;
-   protected String loadAllNonExpiredRowsSql;
-   protected String deleteAllRows;
-   protected String selectExpiredRowsSql;
-   protected String deleteExpiredRowsSql;
-
-   AbstractTableManager(ConnectionFactory connectionFactory, TableManipulationConfiguration config, DbMetaData metaData, Log log) {
+   protected final String identifierQuoteString;
```

Comment:
Cab be private as passing through constructor

Reply:
We are using it here https://github.com/infinispan/infinispan/blob/ce493e14b3/persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/impl/table/OracleTableManager.java#L82